cmd/kola/kola.go

```diff
@@ -237,8 +237,7 @@ func writeProps() error {
 			ImageURL:              kola.PacketOptions.ImageURL,
 		},
 		QEMU: QEMU{
-			Image:   kola.QEMUOptions.DiskImage,
-			Mangled: !kola.QEMUOptions.UseVanillaImage,
```

Comment:
Whoops, just noticed this: the member definition should be dropped too.

Reply:
:recycle: